flake.nix

```diff
@@ -1,18 +1,23 @@
 {
   description = "ocaml-packages-overlay";
 
-  inputs.flake-utils.url = "github:numtide/flake-utils";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
 
-  outputs = { self, flake-utils }: ({
-    overlay = final: prev: (import ./default.nix) final prev;
+  outputs = { self, nixpkgs, flake-utils }: ({
+    overlays.default = final: prev: (import ./default.nix nixpkgs) final prev;
```

Comment:
you don't actually need this, right? it's just an overlay, it can be applied to any `pkgs`. I'd prefer if we could leave `default.nix` as is (it doesn't need to be parameterized over `nixpkgs` (it's the same as `super`)